src/ui/base-page-storyEdit.js

```diff
@@ -222,25 +227,45 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
         return filteredBlocks;
     };
 
-    /* Returns true if a block is determined to be empty, based on the values
+    /**
+     * Returns true if a block is determined to be empty, based on the values
      * listed in blockContentValuesForType. If at least one of those values is
      * truthy, the block is not empty. If the values are empty or otherwise can't
      * be iterated over, then the block is also empty regardless of its contents.
-     * - "block": a single story block (sjrk.storyTelling.block)
-     * - "blockContentValuesForType": an array of values for the block's type
+     *
+     * @param {Object} block - a single story block (sjrk.storyTelling.block)
+     * @param {Object} blockContentValuesForType - an array of values for the block's type
```

Comment:
The type is `{Object}` while the description says "an array". Please double check and fix. This description sentence is not understandable. Please fix too. Thanks.

Reply:
It's a pretty confusing sentence, I agree. I've had a lot of trouble naming this collection succinctly! I'll come up with something :)

Reply:
As for the type, JSDoc [recommends](https://jsdoc.app/tags-type.html) either `{Array.<String>}` or `{String[]}`. I'll use the latter if that's alright

Reply:
Sure.

Reply:
We've also been using `{String[]}` in other projects lately.